app/ui/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -1082,7 +1085,7 @@ private boolean draftIsNotEmpty() {
         return subjectView.getText().length() != 0 ||
                 !recipientPresenter.getToAddresses().isEmpty() ||
                 !recipientPresenter.getCcAddresses().isEmpty() ||
-                !recipientPresenter.getBccAddresses().isEmpty();
+                (recipientPresenter.getBccAddresses().size() - alwaysBccAddrCount) > 0;
```

Comment:
Thanks for PR #4557. Before this change it would just have checked if these alwaysBcc addresses are in the field but _they would have probably been of an other account_, because the addresses wouldn't have changed automatically. So this wouldn't have been any better. But introducing this feature and then checking for correctness, if far the better option, sure. :)